featuretools/primitives/standard/aggregation_primitives.py

```diff
@@ -196,13 +305,27 @@ def n_most_common(x, n=self.number_output_features):
 class AvgTimeBetween(AggregationPrimitive):
     """Computes the average time between consecutive events.
 
-    Note: equivalent to Mean(Diff(time_index)), but more performant
-    """
+    Description:
+        Given a list of datetimes, return the average time
```

Comment:
this description is missing the unit of the return value

Reply:
Fixed.